pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -541,7 +628,8 @@
     "runtime.win7-x64.runtime.native.System.IO.Compression": {
       "StableVersions": [
         "4.0.1"
-      ]
+      ],
+      "BaselineVersion": "4.3.0"
     },
     "runtime.win7-x86.runtime.native.System.Data.SqlClient.sni": {
```

Comment:
@saurabh500 Who did you work with on CoreFX team for those changes? Per my last discussion with @weshaggard on this, there were no changes known that would mandate a creating a new branch in TFS and thus, it was decided to reuse 1.0.0 RTM packages.